UIKitMovie/UIKitMovie/Helpers/KeyChainService.swift

```diff
@@ -0,0 +1,24 @@
+// KeyChainService.swift
+// Copyright © RoadMap. All rights reserved.
+
+import KeychainSwift
+
+/// Сервис шифровки данных
+final class KeyChainService: KeyChainServiceProtocol {
```

Comment:
```suggestion
final class KeyChainService: KeyChainServiceProtocol {
```
Посмотри реализацию у Даниила у него более универсальный сервис